dsl/pace/dsl/dace/sdfg_opt_passes.py

```diff
@@ -42,4 +18,5 @@ def splittable_region_expansion(sdfg: dace.SDFG):
                     "I",
                     "K",
                 ]
-                print("Reordered schedule for", node.label)
+                if verbose:
+                    print("Reordered schedule for", node.label)
```

Comment:
Is there a sort of `logging` library that Pace is using that could be used here?

Reply:
https://docs.python.org/3/library/logging.html

Reply:
Moving to logging.
I think I didn't want to log because sometime the log are "later" to flush during `daint`. But it can't probably be looked into